### ISSUE TITLE
Order Creation: Accessibility → Customer section headings should be marked as headings for VoiceOver

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-ios#6074]
When adding shipping and billing addresses using VoiceOver, the headings like Details and Address are not dictated as headings. Marking these as headings allows VoiceOver users to navigate the screen via heading. 

<img src="https://user-images.githubusercontent.com/198826/152435813-f29a731f-0e43-428d-ad00-0af19e9687d8.png" width="300">
